src/components/Timer/timer.test.ts

```diff
@@ -0,0 +1,37 @@
+// import { Challenge } from '../../model/ChallengeConfiguration'
+/**
+ * import gpx path data
+ * read data from file and get string from data
+ * convert string data to be a TrackPath
+ */
+const testTimer = test('returns time from component', () => {
+  // const defaultChallenge: Challenge = {
```

Comment:
Looks like the contents of this test file are commented out. Maybe you forgot to push some of your code? :test_tube: